Reloj_checador/app/assets/stylesheets/home.scss

```diff
@@ -1,19 +1,19 @@
 @import url('https://fonts.googleapis.com/css?family=Numans');
 html,body{
-    background-image: url('https://mcdn.wallpapersafari.com/medium/37/48/dxjKpR.jpg');
+    background-color: #242236 !important;
     background-size: cover;
     background-repeat: no-repeat;
     height: 100%;
     font-family: 'Numans', sans-serif;
     }
     
-    .container{
+    .container2{
```

Comment:
I'm not a fan of `thing1, thing2, thing3...` naming schemes. How about using a name that tells us more about what the container's purpose instead? Something like `main-container` or `content-container`.

Reply:
Changed the name of the container2 to login-container.